bluebottle/payments_interswitch/tests/test_adapters.py

```diff
@@ -15,7 +15,7 @@
     'MERCHANT_ACCOUNTS': [
         {
             'merchant': 'interswitch',
-            'currency': 'NGN',
+            'currencies': ['NGN'],
```

Comment:
Seems I looked at the wrong branch from server-deployment. I'll close this pull request, and will add a new one that only adds the tests.